projects/arlas-toolkit/src/lib/tools/utils.ts

```diff
@@ -324,3 +331,15 @@ export function hasContributorData(contributor: Contributor): boolean {
     return false;
   }
 }
+
+export function orderAlphabeticallyArlasSearchFields(arlasSearchFieldA: ArlasSearchField, arlasSearchFieldB: ArlasSearchField,){
+  const startLetter = arlasSearchFieldA?.label.trim().toLowerCase();
```

Comment:
it is not startLetter anymore

Reply:
yes we do not need the first letter to sort alphabetically

Reply:
But what you have here is the whole word, not the first letter, and you compare them directly. I am just pointing at a variable that needs to be renamed

Reply:
ahh yes. i ll rename it !